css/main.css

```diff
@@ -1,3 +1,146 @@
-/* Add your custom CSS here */
+* {
+  margin: 0;
+  padding: 0;
+}
+html,
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+html {
+  box-sizing: border-box;
+}
+body {
+  background-color: #e6dacd;
+}
+
+/* Menu Bar */
+.navbar-light {
+  padding-bottom: 100px;
+}
+nav > div > div > a {
```

Comment:
I would recommend using a class here. If your design gets more complicated, you have a risk of having this style applied to different ``<a>`` tags by accident